app/features/contacts/contact-hooks.ts

```diff
@@ -140,18 +144,16 @@ export function useFindContactCandidates(query: string) {
   const contactRepository = useContactRepository();
   const userRef = useUserRef();
 
-  return useQuery({
+  const response = useQuery({
     queryKey: ['search-user-profiles', query],
-    queryFn: async ({ queryKey }) => {
-      const [, search] = queryKey;
```

Comment:
these changes in how I am using query are not related to the bug. I just find the approach I did less verbose so I changed it